Exec/Production/ChallengeProblem/prob.H

```diff
@@ -415,7 +415,8 @@ struct MyProbTagStruct
            cos(prob_parm.cone_angle * constants::PI() / 180.0)) +
         t_loc * t_loc / (prob_parm.r_hole * prob_parm.r_hole);
 
-      if (elrSq < 1.2) {
+      // ProbHi[2] is 0.425 for this problem
+      if (elrSq < 1.2 && z > 0.41) {
```

Comment:
If this works, lets replace with a query to probhi so we don't get confused if we change geom later.

Reply:
we should totally pass in geomdata. Let's not worry about it with this PR.